pages/index.js

```diff
@@ -1,18 +1,99 @@
 import React from 'react';
 
-import ProfileCard from '../components/profile-card';
+import ProfileCard from '../src/components/profile-card';
 import styled from 'styled-components';
 
+
+
 export default function Home() {
   return (
     <Wrapper>
-      나는 홈이야
-      <ProfileCard />
+      <Title>CEOS 프론트엔드 운영진</Title>
+      <CardArea>
+       {Info(profiles)}
+      </CardArea>
```

Comment:
CardArea네이밍 좋네요 👍  몇가지 수정해주시면 더 좋을 것 같습니다!

1. Info라는 함수 이름은 `profile 배열을 받아서, 여러개의 ProfileCard를 반환한다`라는 역할을 충분히 표현하지 못 하는 것 같아요. `getProfileCardList` 같은 이름은 어떨까요?

2. `single`이라는 prop name은 `한 사람의 프로필 정보`라는 느낌이 들지 않는 것 같아요. `profile` 같은 이름은 어떨까요?

2. map처럼 간단한 함수인 경우 return문 안에 그대로 써주셔도됩니다!

```suggestion
      <CardArea>
       {profiles.map(profile =>
         <ProfileCard profile={profile}/>
       )}
      </CardArea>
```